queries.sql

```diff
@@ -71,4 +71,48 @@ GROUP BY neutered;
 /* What is the average number of escape attempts per animal type of those born between 1990 and 2000? */
 SELECT neutered, AVG(escape_attempts) FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-vet_clinic-# GROUP BY neutered;
\ No newline at end of file
+GROUP BY neutered;
+
+/* Foreign Key and JOIN session */
+/* What animals belong to Melody Pond? */
+SELECT A.name, O.full_name
+FROM animals A
+JOIN owners O ON A.owner_id = O.id
+WHERE O.full_name = 'Melody Pond';
+
+/* List of all animals that are pokemon (their type is Pokemon). */
+SELECT A.name, S.name
+FROM animals A
+JOIN species S ON  A.species_id = S.id
+WHERE S.name = 'Pokemon';
+
+/* List all owners and their animals, remember to include those that don't own any animal. */
+SELECT O.full_name, A.name
+FROM owners O
+LEFT JOIN animals A ON O.id = A.owner_id;
+
+/* How many animals are there per species? */
+SELECT S.name, COUNT(S.name)
+FROM species S
+JOIN animals A ON S.id = A.species_id
+GROUP BY S.name;
+
+/* List all Digimon owned by Jennifer Orwell. */
+SELECT O.full_name, A.name, S.name
+FROM animals A
+JOIN owners O ON O.id = A.owner_id
+JOIN species S ON A.species_id = S.id
+WHERE S.name = 'Digimon' AND O.full_name = 'Jennifer Orwell';
+
+/* List all animals owned by Dean Winchester that haven't tried to escape. */
+SELECT A.name, A.escape_attempts, O.full_name
+FROM animals A
+JOIN owners O ON A.owner_id = O.id
+WHERE A.escape_attempts = 0 AND O.full_name = 'Dean Winchester';
+
+/* Who owns the most animals? */
+FROM (SELECT ctn, ct.name AS name
+FROM (SELECT COUNT(O.full_name) AS ctn, O.full_name As name
+FROM owners O
+JOIN animals A ON O.id = A.owner_id
+GROUP BY O.full_name)ct)c2;
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a requirement for this project, please ensure this query returns the name of the person who owns the most animals instead of the number.